### DISCUSSION TITLE
Paid Membership Pro payment gateway

### DISCUSSION THREAD

**Discussion by bitcoinbrisbane** [btcpayserver/btcpayserver#1173] [Plugins & integrations]
At blockchainaustralia.org we use a WP site, with a plugin "paid membership pro".  There is a crypto payment gateway called "GoURL"

I've forked it and will modify to be BTCPayServer instead.  This issue is just to capture that or any integration required.  

https://github.com/bitcoinbrisbane/Bitcoin-Wordpress-Plugin

**Comment by NicolasDorier** [comment]
@bitcoinbrisbane we can maybe also reuse their API on btcpay side?

**Comment by bitcoinbrisbane** [comment]
Yeah I’m exploring all options. Hopefully BTCPayServer can post back to their IPN

**Comment by pavlenex** [comment]
@bitcoinbrisbane any progress on this?

**Comment by bitcoinbrisbane** [comment]
@pajasevi no not yet, if there is some interest i might revisit it.
